### PR TITLE
OEL-2635: List page url alias pattern.

### PR DESCRIPTION
**Pull request by AaronGilMartinez**
### Description

[Insert description here]

### Change log

- Added: New Alias pattern for list pages.
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: modules/oe_showcase_list_pages/oe_showcase_list_pages.post_update.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+use Drupal\oe_bootstrap_theme\ConfigImporter;
+
+/**
+ * Add alias pattern for list pages.
+ */
+function oe_showcase_list_pages_post_update_00001(&$sandbox): void {
+  ConfigImporter::importSingle(
+    'module',
+    'oe_showcase_list_pages',
+    '/config/post_updates/00001',
```
Comment (drishu) [inline modules/oe_showcase_list_pages/oe_showcase_list_pages.post_update.php]: add some representative machine name to  the folder something like 00001_url_alias

File: modules/oe_showcase_list_pages/config/install/pathauto.pattern.list_page_url_alias_pattern.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: list_page_url_alias_pattern
+label: 'List page URL Alias Pattern'
+type: 'canonical_entities:node'
+pattern: 'list/[node:title]'
```
Comment (drishu) [inline modules/oe_showcase_list_pages/config/install/pathauto.pattern.list_page_url_alias_pattern.yml]: the ticket indicated "Add a simple url alias pattern that uses the title." I think that means:
```suggestion
pattern: '[node:title]'
```

**Review by drishu — DISMISSED** [review]

File: modules/oe_showcase_list_pages/oe_showcase_list_pages.post_update.php (hunk 1 above)
Comment (AaronGilMartinez) [reply modules/oe_showcase_list_pages/oe_showcase_list_pages.post_update.php]: Done

File: modules/oe_showcase_list_pages/config/install/pathauto.pattern.list_page_url_alias_pattern.yml (hunk 2 above)
Comment (AaronGilMartinez) [reply modules/oe_showcase_list_pages/config/install/pathauto.pattern.list_page_url_alias_pattern.yml]: Changed

**Review by brummbar — APPROVED** [review]

**Pull request merged**